test/promql/tests/test_notready_pods.yaml

```diff
@@ -0,0 +1,88 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+
+      # Pod is Pending, not unschedulable and its alerted
+      - series: 'kube_pod_status_phase{pod="pod-1", namespace="ns-1", phase="Pending"}'
```

Comment:
We need all phases for all pods - at least for some of the test cases.

Reply:
I've updated the unit test cases now.